ee/localserver/krypto-ec-middleware.go

```diff
@@ -306,15 +316,44 @@ func (e *kryptoEcMiddleware) Wrap(next http.Handler) http.Handler {
 		bhr := &bufferedHttpResponse{}
 		next.ServeHTTP(bhr, newReq)
 
+		// add headers to the response map
+		var responseMap map[string]interface{}
+		bhrBytes := bhr.Bytes()
+		if err := json.Unmarshal(bhrBytes, &responseMap); err != nil {
+			traces.SetError(span, err)
+			e.slogger.Log(r.Context(), slog.LevelError,
+				"unable to unmarshal response",
+				"err", err,
+			)
+			responseMap = map[string]any{
+				"headers": bhr.Header(),
+
+				// the request body was not in json format, just pass it through as "msg"
+				// property of json
+				"msg": string(bhrBytes),
```

Comment:
```suggestion
				"body": string(bhrBytes),
```

_maybe_? I'm really not sure here. I don't think anything we have takes this code path, and I'm not sure what we should aim for.